google-cloud-resourcemanager/src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -302,8 +302,9 @@ public static ProjectListOption fields(ProjectField... fields) {
    * <p>An example of using the read-write-modify pattern is as follows:
    * <pre> {@code
    * Policy currentPolicy = resourceManager.getPolicy("my-project-id");
-   * Policy modifiedPolicy =
-   *     current.toBuilder().removeIdentity(Role.VIEWER, Identity.user("user@gmail.com"));
+   * Policy modifiedPolicy = current.toBuilder()
+   *     .removeIdentity(Role.of("roles/viewer"), Identity.user("user@gmail.com"))
```

Comment:
Same here, prefer shorter version. Also, If I read it correctly and the notion of ["Primitive roles"](https://cloud.google.com/iam/docs/understanding-roles) apply to all, I highly suggest creating shortcut methods for them such as `Role.viewer()`, ...